frontend/src/components/Combat/Combat.tsx

```diff
@@ -31,47 +87,159 @@ export default class Combat extends Component<Props, State> {
             action: move,
             moveData: {
                 item: null,
-                spell: 'fireball',
+                spell: "fireball",
```

Comment:
might be worth setting up something like a prettier config so you all arent changing each other's coding styles back and forth

Reply:
weird, it should be setup already. I'll look into that.